### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by charlesjliu**
updated README with instructions for workflow to contributing. Borrowed from the frontend repo.

### CODE REVIEW

**Comment by nitish24p** [comment]
Can also add the zenhub link as in the FE repo.

File: README.md [hunk 1]
```diff
@@ -29,3 +29,39 @@ docker run -p 3000:80 coding-coach
...
+`git clone git@github.com:YOUR_GITHUB_USERNAME/coding-coach-api.git` (Make sure to replace the URL to your own repository).
+Add the original project as a remote, for this example the name is upstream, feel free to use whatever name you want.
+`git remote add upstream git@github.com:Coding-Coach/coding-coach-api.git`.
+Forking the project will create a copy of that project in your own GitHub account, you will commit your work against your own repository.
+
+### Update your local copy
+In order to update your local environment to the latest version on development, you will have to pull the changes using the upstream repository, for example: git pull upstream development. This will pull all the new commits from the origin repository to your local environment.
+
+### Features/Bugs/Chores
+When working on a new feature, create a new branch feature/something from the development branch, for example `feature/login-form`. Commit your work agains this new branch and push everything to your forked project. Once everything is completed, you should create a PR to the original project. Make sure to add a description about your work and a link to the trello task.
```
Comment (guillermaster) [inline README.md]: There's a small typo in here:
 Commit your work **against**

**Review by guillermaster — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (charlesjliu) [reply README.md]: good catch, fixing!

**Comment by charlesjliu** [comment]
@nitish24p 👍 added

**Review by KaiWedekind — APPROVED** [review]
👌

**Pull request merged**